streams/src/main/java/org/apache/kafka/streams/state/internals/QueryableStoreProvider.java

```diff
@@ -43,20 +43,23 @@ public QueryableStoreProvider(final List<StateStoreProvider> storeProviders,
      * Get a composite object wrapping the instances of the {@link StateStore} with the provided
      * storeName and {@link QueryableStoreType}
      *
+     * @param <T>                The expected type of the returned store
      * @param storeName          name of the store
      * @param queryableStoreType accept stores passing {@link QueryableStoreType#accepts(StateStore)}
-     * @param <T>                The expected type of the returned store
+     * @param includeStandbyAndRecovering     if true, include standbys and recovering stores;
```

Comment:
any reason to swtich to `includeStandbyAndRecovering`? could we keep `queryStaleState`?

Reply:
I had been thinking about it as a directive internally about which stores to bundle, versus an API statement about which consistency level to query at, but upon reflection, even the API is just getting a store, not actually doing a query. I'm standardizing both terms to be the same.